doc/conf.py

```diff
@@ -70,6 +71,16 @@
     "pyvista.ext.plot_directive",
 ]
 
+# Intersphinx mapping
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/dev', None),
+    'scipy': ('https://docs.scipy.org/doc/scipy/reference', None),
+    'numpy': ('https://numpy.org/devdocs', None),
+    'matplotlib': ('https://matplotlib.org/stable', None),
+    'imageio': ('https://imageio.readthedocs.io/en/stable', None),
+    'pandas': ('https://pandas.pydata.org/pandas-docs/stable', None),
+    'pytest': ('https://docs.pytest.org/en/stable', None),
+}
```

Comment:
Neat, I've been wondering about this! But now we need a major refactor to use these names, or add `np` as an alias here, or do something like https://github.com/numpy/numpydoc/commit/e93218e1e656eb6970bf77d076b5625166c3efeb#diff-185fc63a0740bab8e3f4da4389ba5bf12230004ba06d5878d9d5ad962ac68222R64-R81 (not present in https://github.com/numpy/numpydoc/blob/main/doc/install.rst) to define aliases:

![Screenshot from 2021-07-12 22-41-23](https://user-images.githubusercontent.com/17914410/125353062-5a868e80-e362-11eb-8dd1-fc862fc70670.png)
(Note the "string" at the top and the "floats" at hte bottom which aren't links; latter should definitely be `float`, singular).

On a similar note, is there a way to render return types as links as well?
![Screenshot from 2021-07-12 22-44-14](https://user-images.githubusercontent.com/17914410/125353336-b5b88100-e362-11eb-89c3-bc3861bdaf11.png)


Perhaps we just need `numpydoc_xref_param_type = True`? https://github.com/numpy/numpydoc/blob/main/doc/install.rst

Reply:
Since we're not using `numpydoc`, those parameters won't work.  I like how Sphinx renders a little bit better, so I don't really want to switch.  If we want our return types to rerun as links, spelling out our return types correctly goes a long way.  I think part of the issue is np vs numpy.

Reply:
Ah, OK, sorry. It's not clear to me at all how these doc things relate to one another, I just saw the `numpydoc_use_plots = True` in `doc/conf.py`.

No, I edited that return value to `numpy.ndarray` first. See also the `list` that isn't a link. So it's both the `np` _and_ the fact that they are return types.

Reply:
`numpydoc_use_plots = True` should be removed.  I left that in during the plot directive PR when trying it out unsuccessfully.

I've seen link for other numpy types, not sure why returns are giving us grief.

Reply:
Just to be sure we're on the same page: it's not just numpy types not linking in returns. Look at the `list` type of `cpos` in my last screenshot.

Reply:
Saw that as well.  I really hope we don't need numpydoc to get that to work.

Reply:
I don't think this is important enough to merit major changes in doc building. If there's an easy way to do it then great; otherwise just leave it as is. The intersphinx addition is already really nice.

Reply:
This ended up being hard to figure out.  First, our docs do return types with links, but only if we have a single return type.  With multiple return types, all bets are off. However, if you manually specify the class with the `:class:` directive, it always works.

Another note, I've changed the format of the returns to be like the parameter return type (description and type inline).  This breaks return type linking even with a single return type, but since we're going to convert these over I figure it's fine.

Change is the following in `conf.py`
```py
napoleon_use_rtype = False
```